sbi/analysis/plot.py

```diff
@@ -976,3 +977,110 @@ def _get_default_opts():
         },
         "title_format": {"fontsize": 16},
     }
+
+
+def plot_p_values_local_test(
```

Comment:
maybe delete this and just take a look at local p values and plot_significant_obs in pp_plots_local_coverage